xml/System.Reflection.Metadata/MetadataReader.xml

```diff
@@ -2133,8 +2138,8 @@
         <ReturnType>System.Reflection.Metadata.MetadataReaderOptions</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Options passed to the constructor.</summary>
```

Comment:
what do these options represent? And it should be:

Gets the...

Reply:
@nguerrera @tmat What would be a good description for this?